os/gfile/gfile_z_example_contents_test.go

```diff
@@ -0,0 +1,236 @@
+// Copyright GoFrame Author(https://goframe.org). All Rights Reserved.
+//
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file,
+// You can obtain one at https://github.com/gogf/gf.
+
+package gfile_test
+
+import (
+	"fmt"
+
+	"github.com/gogf/gf/v2/os/gfile"
+)
+
+func ExampleGetContents() {
+	// init
+	var (
+		fileName = "gflie_example.txt"
+		tempDir  = gfile.TempDir("gfile_example_content")
+		tempFile = gfile.Join(tempDir, fileName)
+	)
+
+	// write contents
+	gfile.PutContents(tempFile, "goframe example content")
+
+	// read contents
```

Comment:
It reads and returns the file content as string.
It returns empty string if it fails reading, for example,  with permission or IO error.